src/idl_parser.cpp

```diff
@@ -723,6 +725,10 @@ CheckedError Parser::ParseField(StructDef &struct_def) {
     // This will cause an error if the root type of the nested flatbuffer
     // wasn't defined elsewhere.
     LookupCreateStruct(nested->constant);
+
+    // Keep a pointer to StructDef in attribute to simplify re-use later
+    auto nested_qualified_name = namespaces_.back()->GetFullyQualifiedName(nested->constant);
+    nested->type.struct_def = structs_.Lookup(nested_qualified_name);
```

Comment:
Fair enough. Also, storing it in `nested_flatbuffer` requires less lookups in the later run, no need to actually fetch the attribute. Change has been committed and pushed to the pull request.